sql/understanding-the-query-execution-plan.md

```diff
@@ -28,6 +28,45 @@ TiDB 优化器会根据当前数据表的实际情况来选择最优的执行计
 | task          | 当前这个 operator 属于什么 task。目前的执行计划分成为两种 task，一种叫 **root** task，在 tidb-server 上执行，一种叫 **cop** task，并行的在 TiKV 上执行。当前的执行计划在 task 级别的拓扑关系是一个 root task 后面可以跟许多 cop task，root task 使用 cop task 的输出结果作为输入。cop task 中执行的也即是 TiDB 下推到 TiKV 上的任务，每个 cop task 分散在 TiKV 集群中，由多个进程共同执行。 |
 | operator info | 每个 operator 的详细信息。各个 operator 的 operator info 各有不同，详见 [Operator Info](#operator-info)。                   |
 
```

Comment:
One comment not related to this PR. In the first section of this file, we have:

> EXPLAIN 可以和 SELECT，DELETE，INSERT，REPLACE，以及 UPDATE 语句一起使用；

maybe we should remove `INSERT` and `REPLACE` in this PR together.

```
mysql> explain insert into t1 values(1,1);
ERROR 1105 (HY000): Unsupported type *core.Insert
mysql> explain replace into t1 values(1,1);
ERROR 1105 (HY000): Unsupported type *core.Insert
```

Reply:
Uh, they are bugs, I think we should fix it 😂

Reply:
I'll remove them in this PR, let's add them back once the bugs are fixed.